src/plugins/inspector/public/index.scss

```diff
@@ -1 +1,7 @@
-@import 'views/index'
+@import 'views/index';
+
+.insFlyoutBody {
+  .euiFlyoutBody__overflowContent {
+    height: 100%;
```

Comment:
Why is this necessary? Targeting EUI classes is usually not a good idea. There may be a better way to fix whatever issue you ran into.

Reply:
@cchaos I have not found a better way to achieve my goal. The problem is that I want to set the height to 100% for the `DataGrid` / `EuiCodeEditor` component. Without these changes, it looks like this:
![image](https://user-images.githubusercontent.com/20072247/87557033-19608d80-c6c0-11ea-909f-9c73b7cd91f0.png)


I would really appreciate if you could help me do it in more professionally way.